assets/xml/overlays/ovl_En_Bom.xml

```diff
@@ -1,9 +1,20 @@
 <Root>
+    <!-- Assets for the Powder Keg -->
     <File Name="ovl_En_Bom" BaseAddress="0x80870DB0" RangeStart="0x2160" RangeEnd="0x3610">
-        <DList Name="ovl_En_Bom_DL_80872F98" Offset="0x21E8"/>
-        <DList Name="ovl_En_Bom_DL_80873020" Offset="0x2270"/>
-        <DList Name="ovl_En_Bom_DL_80873040" Offset="0x2290"/>
-        <DList Name="ovl_En_Bom_DL_80873CA0" Offset="0x2EF0"/>
-        <DList Name="ovl_En_Bom_DL_808742F8" Offset="0x3548"/>
+        <!-- Texture and DisplayLists for the Powder Keg's fuse -->
+        <Texture Name="gPowderKegFuseTex" OutName="powder_keg_fuse" Format="i8" Width="2" Height="2" Offset="0x21E0"/>
+        <DList Name="gPowderKegFuseMaterialDL" Offset="0x21E8"/>
+        <DList Name="gPowderKegFuseModel1DL" Offset="0x2270"/>
+        <DList Name="gPowderKegFuseModel2DL" Offset="0x2290"/>
+
+        <!-- Textures and DisplayList for the Powder Keg -->
+        <Texture Name="gPowderKegTLUT" OutName="powder_keg_tlut" Format="rgba16" Width="16" Height="16" Offset="0x22B0"/>
+        <Texture Name="gPowderKegTex" OutName="powder_keg" Format="ci8" Width="32" Height="32" Offset="0x24B0"/>
+        <DList Name="gPowderKegDL" Offset="0x2EF0"/>
+
+        <!-- Textures and DisplayList for the Goron Skull on the side of the Powder Keg -->
+        <Texture Name="gPowderKegGoronSkullTLUT" OutName="powder_keg_goron_skull_tlut" Format="rgba16" Width="16" Height="16" Offset="0x3048"/>
+        <Texture Name="gPowderKegGoronSkullTex" OutName="powder_keg_goron_skull" Format="ci8" Width="16" Height="32" Offset="0x3248"/>
+        <DList Name="gPowderKegGoronSkullDL" Offset="0x3548"/>
```

Comment:
Is it actually a Goron Skull? I thought it was a generic skull. We should make sure these names are synced with the `GI` version in my `z_draw` PR, although it looks pretty similar.
https://github.com/engineer124/mm/blob/z_draw/assets/xml/objects/object_gi_bigbomb.xml

Reply:
Yeah, it's subtly different from the GI version, but it's similar enough.
GI version:
![image](https://user-images.githubusercontent.com/12245827/180046837-c2d53c84-abcc-47c8-9519-d65779f46d07.png)

En_Bom version:
![image](https://user-images.githubusercontent.com/12245827/180046903-6806ff42-e59b-4ffa-9e8a-ca4b526386c8.png)

As for why I called it a "Goron Skull", it doesn't look really human. It has eyes that are really far apart and a pointed head. It's even more obviously a Goron (in my opinion) in MM3D:
![image](https://user-images.githubusercontent.com/12245827/180047091-28e1550d-36a5-4a4b-a99b-23a78a1434ec.png)